index.html

```diff
@@ -16,13 +16,45 @@
           />
         </a>
 
-        <a href="">
-          Поиск
-        </a>
+        <form action="https://echo.htmlacademy.ru" method="post">
```

Comment:
```suggestion
        <form action="https://echo.htmlacademy.ru" method="get">
```

поиск обычно гет запрос